### DISCUSSION TITLE
Magit v3.0.0 released

### DISCUSSION THREAD

**Discussion by tarsius** [magit/magit#4391] [Announcements]
Finally, a release!

* [Announcement](https://emacsair.me/2021/05/25/magit-3.0/)
* [Release Notes](https://github.com/magit/magit/blob/master/Documentation/RelNotes/3.0.0.org)

<hr /><em>This discussion was created from the release <a href='https://github.com/magit/magit/releases/tag/v3.0.0'>v3.0.0</a>.</em>

**Comment by ryukinix** [comment]
Wonderful!

**Comment by aguynamedben** [comment]
I love magit!!! thank you so much
